packages/AlertDialog/index.tsx

```diff
@@ -28,122 +29,109 @@ const ModalButton = styled.TouchableOpacity`
-        {
-          borderRadius: 4,
-        },
-        styles?.modalContainer,
-      ],
-    };
-
     const renderAlert = (
-      <ModalButton onPress={() => onPress(true)}>
-        <ButtonText color={color}>OK</ButtonText>
+      <ModalButton style={styles?.button} onPress={() => onPress(true)}>
+        <ButtonText style={styles?.buttonText}>OK</ButtonText>
       </ModalButton>
     );
 
     const renderConfirm = (
       <>
-        <ModalButton onPress={() => onPress(false)}>
-          <ButtonText color={color}>CANCEL</ButtonText>
+        <ModalButton style={styles?.button} onPress={() => onPress(false)}>
+          <ButtonText style={styles?.buttonText}>CANCEL</ButtonText>
         </ModalButton>
-        <ModalButton onPress={() => onPress(true)}>
-          <ButtonText color={color}>OK</ButtonText>
+        <ModalButton style={styles?.button} onPress={() => onPress(true)}>
+          <ButtonText style={styles?.buttonText}>OK</ButtonText>
         </ModalButton>
       </>
     );
 
     const renderPrompt = (
       <>
         <ModalButton
+          style={styles?.button}
           onPress={() => {
             onPress(null);
             setInput('');
           }}>
-          <ButtonText color={color}>CANCEL</ButtonText>
+          <ButtonText style={styles?.buttonText}>CANCEL</ButtonText>
         </ModalButton>
         <ModalButton
+          style={styles?.button}
           onPress={() => {
             onPress(input);
             setInput('');
           }}>
-          <ButtonText color={color}>OK</ButtonText>
+          <ButtonText style={styles?.buttonText}>OK</ButtonText>
         </ModalButton>
       </>
     );
 
     return (
-      <>
-        <Modal
-          isOpen={isOpen}
-          style={modalStyles.modal}
-          position={'center'}
-          ref={ref}
-          backdropOpacity={backdropOpacity}
-          backdropPressToClose={backdropPressToClose}>
-          <ContentContainer>
-            {children}
-            {type === 'prompt' && (
-              <ModalInput value={input} onChangeText={setInput} color={color} />
-            )}
-          </ContentContainer>
-          <ButtonContainer>
-            {type === 'alert' && renderAlert}
-            {type === 'confirm' && renderConfirm}
-            {type === 'prompt' && renderPrompt}
-          </ButtonContainer>
-        </Modal>
-      </>
+      <Modal
+        isOpen={isOpen}
+        style={[{borderRadius: 4}, styles.modal]}
+        position={'center'}
+        ref={ref}
+        backdropOpacity={backdropOpacity}
+        backdropPressToClose={backdropPressToClose}>
+        <ContentContainer>
+          {children}
+          {type === 'prompt' && (
+            <ModalInput value={input} onChangeText={setInput} />
+          )}
+        </ContentContainer>
+        <ButtonContainer>
+          {type === 'alert' && renderAlert}
+          {type === 'confirm' && renderConfirm}
+          {type === 'prompt' && renderPrompt}
```

Comment:
When changing render function correctly you might want to call it here `renderAlert()`.